release_notes/ocp_3_7_release_notes.adoc

```diff
@@ -1195,6 +1195,14 @@ perform new tasks.
 See xref:../cli_reference/extend_cli.adoc#cli-reference-extend-cli[Extending the
 CLI] for information on how to install and write extensions for the CLI.
 
+[[ocp-37-chained-builds]]
+==== Chained Builds
+
+In {product-title} 3.7,
+xref:../dev_guide/builds/advanced_build_operations.html#dev-guide-chaining-builds[Chaining
+Builds] is a better approach for producing runtime-only
+application images, and fully replaces the Extended Builds feature.
+
```

Comment:
Do users know what chain builds are?

Reply:
Good point, @mburke5678! Chained Builds have been in the documentation for a while, but I went ahead and included the benefits listed in the original announcement. Do you think that is sufficient?